lib/bloc/account/account_bloc.dart

```diff
@@ -353,22 +353,13 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
 
     // Apply payment type filters, if there's any
     if (paymentFilters.filters != null && paymentFilters.filters != sdk.PaymentTypeFilter.values) {
+      final filters = paymentFilters.filters!;
```

Comment:
I meant having this before the condition itself. i.e.
```
final paymentTypeFilters = paymentFilters.filters;
if (paymentTypeFilters != null && paymentTypeFilters != sdk.PaymentTypeFilter.values) {
  filteredPayments = filteredPayments.where((paymentMinutiae) {
    return paymentTypeFilters.any((filter) => filter.name == paymentMinutiae.paymentType.name);
  }).toList();
}
```

Reply:
Ahh true, my bad